recipes/orange3/meta.yaml

```diff
@@ -9,8 +9,6 @@ source:
 
 build:
   number: 0
-  script:
-    - python setup.py install
```

Comment:
Thanks

I have updated the script command to be the same that was used in build.sh. It uses record=null, should I change it to record.txt as you have suggested?